lib/android/src/main/java/com/airbnb/android/react/navigation/ScreenCoordinator.java

```diff
@@ -38,17 +36,36 @@
   static final String EXTRA_PAYLOAD = "payload";
   private static final String TRANSITION_GROUP = "transitionGroup";
 
+  enum PresentAnimation {
+    Modal(R.anim.slide_up, R.anim.delay, R.anim.delay, R.anim.slide_down),
+    Push(R.anim.slide_in_right, R.anim.slide_out_left, R.anim.slide_in_left, R.anim.slide_out_right),
+    Fade(R.anim.fade_in, R.anim.fade_out, R.anim.fade_in, R.anim.fade_out);
```

Comment:
wonder if we shouldn't make this an enum so that people can implement their own?

For instance, we could just have a `PresentAnimation` class and our navigation implementation could just have an overridable `PresentAnimation getModalAnimation()` and a `PresentAnimation getPushAnimation()` and `PresentAnimation getFadeAnimation()` so people can customize it?